File: workspace/prime.js
var isnprime = [];
var prime = [], cnt = 0;
const MAXN = 10000000;
for (var i = 1; i <= MAXN; ++i) isnprime[i] = 0;
isnprime[1] = 1;
for (var i = 2; i <= MAXN; ++i) {
  if (!isnprime[i]) prime[++cnt] = i;
  for (var j = 1; j <= cnt; ++j) {
    if (i * prime[j] > MAXN) break;
    isnprime[i * prime[j]] = 1;
    if (i % prime[j] == 0) break;
  }
}
for (var i in prime) {
  console.log(prime[i]);
}

File: workspace/prime.cpp
#include <cstdio>
using namespace std;
const int MAXN = 600005;
bool isnprime[MAXN];
int prime[MAXN], cnt;
void Get_Prime(int n) {
  isnprime[1] = 1;
  for (int i = 1; i <= n; i++) {
    if (!isnprime[i])
      prime[++cnt] = i;
    for (int j = 1; j <= cnt; j++) {
      if (i % prime[j] == 0) break;
      isnprime[i * prime[j]] = 1;
    }
  }
}
int main() {
  Get_Prime(10000);
  for (int i = 1; i <= 100; ++i) {
    printf ("%d\n", prime[i]);
  }
}